### DISCUSSION TITLE
Naming things: change website to tracker

### DISCUSSION THREAD

**Discussion by jesperordrup** [umami-software/umami#2574] [General]
### Describe the feature or enhancement

Teams and users are clear but it a bit misleading to say that you create websites

**Comment by franciscao633** [comment]
Can you elaborate? Sorry I'm not understanding the issue.

**Comment by jesperordrup** [comment]
when you in umami say "add website" - you're not adding a website. You are creating a tracker / script for your website. 

"create tracker"
"create website tracker"

**Comment by thrzl** [comment]
i believe it’s that you’re adding a website to umami
